### PR TITLE
feat(member-invites): Allow changing request role / teams

### PR DESCRIPTION
**Pull request by evanpurkhiser**
image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/organizationMembers/inviteRequestRow.tsx [hunk 1]
```diff
@@ -62,11 +73,34 @@ const InviteRequestRow = ({
           </Tooltip>
         )}
       </div>
-      <div>{inviteRequest.roleName}</div>
+      <div>
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organizationMembers/inviteRequestRow.tsx]: Are these divs necessary?

File: src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx [hunk 2]
```diff
@@ -114,18 +118,26 @@ class OrganizationMembersWrapper extends AsyncView<Props> {
     return count ? count.toString() : null;
   }
 
-  updateRequestList = (id: string) =>
+  removeAccessRequest = (id: string) =>
     this.setState(state => ({
-      requestList: state.requestList.filter(({id: existingId}) => existingId !== id),
+      requestList: state.requestList.filter(m => m.id !== id),
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx]: I'm not sure if the single character name is better than the previous destructure. Can we give it a better name?

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/settings/organizationMembers/inviteRequestRow.tsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/organizationMembers/inviteRequestRow.tsx]: I'll fix this up in a separate PR in a min.

File: src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx (hunk 2 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx]: Yeah I'll fix this up in a separate PR, sounds good.

**Pull request merged**